google-map.html

```diff
@@ -371,7 +371,7 @@
       if (this.map) {
         return; // already initialized
       }
-      if (!(window.google && window.google.maps)) {
+      if (this.$.api.libraryLoaded !== true) {
```

Comment:
Is that really true though? iron-jsonp-library fires sets that property: https://github.com/PolymerElements/iron-jsonp-library/blob/master/iron-jsonp-library.html#L35

If we really want to prevent a race condition with the maps api library loading its modules, we need to wait for `google.maps.Map` to be defined. If the lib lazy attached `Map` to `google.maps`, using `libraryLoaded` doesn't help.